game/scripts/vscripts/Sections/Chapter4/SectionWards.ts

```diff
@@ -172,8 +231,8 @@ function orderFilter(event: ExecuteOrderFilterEvent): boolean {
         return true;
     }
 
-    if (event.order_type === UnitOrder.DROP_ITEM || event.order_type === UnitOrder.MOVE_ITEM || event.order_type === UnitOrder.CAST_TOGGLE) {
-        displayDotaErrorMessage("Dropping, moving or toggling your items is disabled during this section.")
+    if (event.order_type === UnitOrder.DROP_ITEM || event.order_type === UnitOrder.MOVE_ITEM || event.order_type === UnitOrder.CAST_TOGGLE || event.order_type === UnitOrder.SET_ITEM_COMBINE_LOCK) {
+        displayDotaErrorMessage("Dropping, moving, toggling, lock or combine items is disabled during this section.")
```

Comment:
The error message's probably too long. Consider grouping it into something more straightforward, or split the if into different ifs with different messages for each case to make it clear.

Also, grammar: "locking or combining items.."

Reply:
Seperated.